filterLists.py

```diff
@@ -1,6 +1,7 @@
-import pandas as pd
-import re
 from os import path
+import csv
+
+DELIMITER = '\n\n'      # individual bugs in each file would need to be separated by two new lines [YOU CAN CHANGE DELIMITER IF YOU WNAT]
```

Comment:
This eventually shouldn't be in your code, but should rather be read from a config file. (A project is not going to want to hunt for this in your code).